server/Cargo.toml

```diff
@@ -15,27 +15,28 @@ publish = true
 
 [dependencies]
 futures-util = { version = "0.3.14", default-features = false, features = ["io", "async-await-macro"] }
-jsonrpsee-types = { workspace = true }
```

Comment:
did you sort the dependencies here or why is this changed?

Reply:
Yes, this is automatically done by my vscode extension if I change the `Cargo.toml`.

Reply:
dq: that sounds useful, what extension are you using?

Reply:
https://taplo.tamasfe.dev/